Skeleton/include/Volunteer.h

```diff
@@ -10,6 +10,8 @@ using std::vector;
 class Volunteer {
     public:
         Volunteer(int id, const string &name);
+        Volunteer(int id, const string &name, int completedOrderId, int activeOrderId);
```

Comment:
why is it for?

Reply:
No need to, forgot to delete it.
resolved in my new branch